bedrock/firefox/templates/firefox/releases/aurora-notes.html

```diff
@@ -4,7 +4,7 @@
 
 {% extends "firefox/releases/release-notes.html" %}
 
-{% block gtm_page_id %}data-gtm-page-id="/firefox/android/auroranotes/"{% endblock %}
+{% block gtm_page_id %}data-gtm-page-id="/firefox/auroranotes/"{% endblock %}
```

Comment:
Is changing this intentional? The URL in prod seems to be `/firefox/android/{% version %}/auroranotes/`, as far as i can tell?

Reply:
Wow, that is a can of worms.

Both https://www.mozilla.org/en-US/firefox/android/57.0/auroranotes/ and https://www.mozilla.org/en-US/firefox/android/57.0/releasenotes/ seem to work but if you view source on the auroranotes URL you can see that the gtm-page-id indicates that it is using the release notes template.

There is no aurora version of Firefox for Android AFAIK. And the [Aurora pages](https://www.mozilla.org/en-US/firefox/32.0a2/auroranotes/) are being incorrectly identified as Android ones right now.

I suspect the android aurora notes are the product of a configured URL redirect? But I will follow up with @pmac to try to sort this out.

Reply:
How did you find the Android aurora notes URLs?

Reply:
> How did you find the Android aurora notes URLs?

I'm not sure you can navigate to them directly any more. I think Aurora only exists on android now, since Aurora became Dev Edition on desktop?

Reply:
Yeah. The aurora situation remains weird. Aurora is no longer a thing for any browser, but the old aurora release notes should stick around, so we do need to keep the aurora URLs for the aurora releases. The last Fx Android Aurora I see in release notes is 54.0a2:

https://www.mozilla.org/en-US/firefox/android/54.0a2/auroranotes/
https://github.com/mozilla/release-notes/blob/master/releases/firefox-for-android-54.0a2-aurora.json

Reply:
So there is no Android Aurora 57 but there is a page for it: https://www.mozilla.org/en-US/firefox/android/57.0/auroranotes/ :(

Reply:
Oh fun, it works with Desktop release too: https://www.mozilla.org/en-US/firefox/58.0/auroranotes/

Reply:
I poked for 15 minutes and now I need to move onto something else.

For Android and Desktop versions that do not have aurora versions the template being used is the release template, So, the URL is deceptive but the content is accurate. 

I will add an Android Aurora test case to my tests and make sure the content is accurate when an aurora version exists, not sure what else to do. @pmac is this worthy of a bug?

Reply:
I filed https://github.com/mozilla/bedrock/issues/6930 to fix the gtm_page_id problem but that doesn't solve the fact that the URLs resolve when they should probably 404 or forward.

Reply:
Maybe worthy of a bug. We should probably redirect `/auroranotes/` to `/releasenotes/` when it's not aurora, but I'm hoping that traffic to a non-aurora release at the former URL is extremely low and maybe we can just not worry about it?

Reply:
Yeah, I'm going to vote for not worrying about it. No one should see this unless they're typing the URLs in themselves.